PennyMe/StatisticsViewController.swift

```diff
@@ -0,0 +1,273 @@
+            text = "Collector status: Gold!"
+            checkPopupNeeded(userSawLast: userSawLast, aboveCount: goldCutoff, statusName: "Gold",  furtherText: "Keep on collecting to get a second crown at \(goldproCutoff) visited machines.")
+        }
+        else if visitedCount >= silverCutoff{
+            color = silverColor
+            text = "Collector status: Silver!"
+            checkPopupNeeded(userSawLast: userSawLast, aboveCount: silverCutoff, statusName: "Silver", furtherText: "Keep on collecting to reach Gold status at \(goldCutoff) visited machines.")
+        }
+        else if visitedCount >= bronzeCutoff {
+            color = bronzeColor
+            text = "Collector status: Bronze!"
+            checkPopupNeeded(userSawLast: userSawLast, aboveCount: bronzeCutoff, statusName: "Bronze", furtherText: "Keep on collecting to reach Silver status at \(silverCutoff) visited machines.")
+        }
+        
+        // add second crown
+        if visitedCount >= goldproCutoff {
+            secondCrown.image = UIImage(systemName:"crown.fill")
+            checkPopupNeeded(userSawLast: userSawLast, aboveCount: goldproCutoff, statusName: "Gold Pro", furtherText: "Keep on collecting to get a third crown at \(goldlegendCutoff) visited machines.")
+            secondCrown.tintColor = goldColor
+            text = "Collector status: Gold Pro!"
+        }
+        
+        // add third crown
+        if visitedCount >= goldlegendCutoff {
+            thirdCrown.image = UIImage(systemName:"crown.fill")
+            checkPopupNeeded(userSawLast: userSawLast, aboveCount: goldlegendCutoff, statusName: "Gold Legend", furtherText: "")
+            thirdCrown.tintColor = goldColor
+            text = "Collector status: Gold Legend!"
+        }
+        
+        // update variable indicating what status the user has seen
+        UserDefaults.standard.set(visitedCount, forKey: "userSawLast")
+        UserDefaults.standard.synchronize()
+        
+        crownImage.tintColor = color
+        
+        showPercentSwitch.onTintColor = UIColor.black
+        showPercentSwitch.addTarget(self, action: #selector(setBarChartType), for: .valueChanged)
+        
+        byCountryLabel.font = UIFont.systemFont(ofSize: 20, weight: .bold)
+        
+        titleLabel.textColor = .gray
+        titleLabel.text = text
+        titleLabel.numberOfLines = 0
+        titleLabel.font = UIFont.systemFont(ofSize: 20, weight: .bold)
+        titleLabel.translatesAutoresizingMaskIntoConstraints = false
+        
+        numberLabel.font = UIFont.systemFont(ofSize: 60, weight: .bold)
+        numberLabel.shadowColor = UIColor.gray // Shadow color
+        numberLabel.shadowOffset = CGSize(width: 2, height: 2)
+        numberLabel.textColor = color
+        numberLabel.text = "\(visitedCount)"
+        
+        totalMachinesLabel.numberOfLines = 0
+        totalMachinesLabel.text = "machines visited"
+        totalLabel.text = "/ \(totalMachines)"
```

Comment:
Out of curiosity, why do we have access to the `totalMachines` variable here even though it is only defined in the ViewController?